context.go

```diff
@@ -295,6 +303,14 @@ func (c *Context) GetInt64(key string) (i64 int64) {
 	return
 }
 
+// GetInt64 returns the value associated with the key as an unsigned integer.
```

Comment:
comment typo error: `GetInt64` to `GetUint64`

Reply:
Thx, I fixed this.